### PR TITLE
Bootstrap needs cli subdirectory to exist

### PR DESCRIPTION
**Pull request by JeremyKuhne**
@weshaggard @danmosemsft

### CODE REVIEW

File: bootstrap/bootstrap.ps1 [hunk 1]
```diff
@@ -45,6 +45,11 @@ Invoke-WebRequest "https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/script
 $rootCliVersion = Join-Path $RepositoryRoot ".cliversion"
 $dotNetCliVersion = Get-Content $rootCliVersion
 
+if (-Not (Test-Path $CliLocalPath))
```
Comment (weshaggard) [inline bootstrap/bootstrap.ps1]: Do we have the same issue in bootstrap.sh?

**Comment by weshaggard** [comment]
@crummel

File: bootstrap/bootstrap.ps1 (hunk 1 above)
Comment (JeremyKuhne) [inline bootstrap/bootstrap.ps1]: Probably- I'll follow up. This won't pass CI without the fix as it pulls directly from the repo. I assume I can put this in?

Comment (JeremyKuhne) [inline bootstrap/bootstrap.ps1]: We should probably have this repo use the local script instead of pulling from the net. Thoughts?

Comment (crummel) [inline bootstrap/bootstrap.ps1]: We could do an override environment variable similar to BUILDTOOLS_SOURCE.

**Comment by crummel** [comment]
Looks good to me.  I do think the Linux version will have the same issue.

**Comment by crummel** [comment]
Shell script change looks good too, thanks for the fix.

File: bootstrap/bootstrap.ps1 (hunk 1 above)
Comment (weshaggard) [inline bootstrap/bootstrap.ps1]: I would prefer we build in the same way other repo's build unless there is a strong reason.

**Pull request merged**